SCClassLibrary/Common/Streams/Rest.sc

```diff
@@ -1,56 +1,23 @@
-Rest {
-	var <>dur = 1;
-	*new { |dur(1)|
-		^super.newCopyArgs(dur)
-	}
-	// for use by Pfunc and Pfuncn
-	*processRest { |inval|
-		inval.put(\isRest, true);
-		^1
-	}
-	processRest { |inval|
-		inval.put(\isRest, true);
-		^dur.value(inval)
-	}
-	// for use e.g. in ListPatterns
-	*embedInStream { |inval|
-		^this.processRest(inval).yield;  // the actual return value is the next inval
-	}
-	embedInStream { |inval|
-		^this.processRest(inval).yield;
-	}
-	*asStream {
-		^Routine({ |inval|
-			loop {
-				inval = this.embedInStream(inval);
-			}
-		})
-	}
-	asStream {
-		^Routine({ |inval|
-			loop {
-				inval = this.embedInStream(inval);
-			}
-		})
+
+Rest : Operand  {
+
+	*new { |value = 1|
+		^super.newCopyArgs(value.dereferenceOperand).unwrapBoolean
```

Comment:
Also, since my comment was read as suggesting that Brian is personally holding things up... nah, it's a 2 month old review comment that never got closed and had no discussion since then. If anything is holding it up, it would be the inattention to the comment rather than the comment itself. Sorry if my wording wasn't clear.